src/lib/core/CHIPConfig.h

```diff
@@ -2651,6 +2651,46 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_MAX_GROUP_NAME_LENGTH 16
 #endif
 
+/**
+ * @def CHIP_CONFIG_ACCESS_CONTROL_MAX_REQUIRED_PRIVILEGE_OVERRIDES_FOR_READ_ATTRIBUTE
+ *
+ * Defines the maximum number of overrides for the read attribute operation in the
+ * required privilege code.
+ */
+#ifndef CHIP_CONFIG_ACCESS_CONTROL_MAX_REQUIRED_PRIVILEGE_OVERRIDES_FOR_READ_ATTRIBUTE
+#define CHIP_CONFIG_ACCESS_CONTROL_MAX_REQUIRED_PRIVILEGE_OVERRIDES_FOR_READ_ATTRIBUTE 8
+#endif
```

Comment:
Let's remove all these constants

Reply:
Ah man I did but the merge (someone rebased this PR) bfought them back... grr...